wormhole-connect/package.json

```diff
@@ -31,6 +31,7 @@
     "@reduxjs/toolkit": "^1.9.1",
     "@solana/wallet-adapter-wallets": "^0.19.25",
     "@solana/web3.js": "^1.73.0",
+    "@wormhole-foundation/sdk": "^0.5.3-beta.6",
     "@wormhole-foundation/sdk-definitions": "^0.5.2-beta.0",
```

Comment:
probably should just rm `sdk-definitions` if installing sdk, also means no need to update versions in multiple deps

Reply:
WILL DO IN FOLLOWUP PR